### PR TITLE
added a paragraph of text

### PR DESCRIPTION
**Pull request by allisonvilla**
Added a paragraph of text!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,5 +8,9 @@
 </head>
 <body>
     <h1>Welcome to our ❄ Cool Website ❄</h1>
+    <p>Cupcake ipsum dolor sit amet jelly-o pie. Cookie halvah candy canes pastry bear claw sweet. Powder gummies croissant
```
Comment (sylviaraposo) [inline index.html:11]: I need your lorem generator! 🧁 I see no issues with Cookie halvah candy canes pastry bear claw sweet

**Review by sylviaraposo — CHANGES_REQUESTED** [review]
Look's good! Needs more cupcake or cats.

File: index.html (hunk 1 above)
Comment (allisonvilla) [reply index.html:11]: I have added the cats

**Comment by allisonvilla** [comment]
Made requested changes

**Review by sylviaraposo — APPROVED** [review]
Love it! 🐱🐱 All good to merge! 🧁

**Pull request merged**